Resources/Prototypes/Entities/Clothing/OuterClothing/hardsuits.yml

```diff
@@ -139,13 +139,14 @@
     highPressureMultiplier: 0.5
     lowPressureMultiplier: 1000
   - type: ExplosionResistance
-    damageCoefficient: 0.3
+    damageCoefficient: 0.7
```

Comment:
```suggestion
    damageCoefficient: 0.4
```

Reply:
"Miner" hardsuit. Miner. Miners work with explosives and protection against them is a must. Weak-ass protection won't do. May as well remove it, since you know, "miners work with weak and controlled explosions" (70 damage per seismic and there are also mines at some wrecks), since they won't get caught in a blast anyway. 

Protection against wildlife comes with heavy-ass plating (that also reduces MS by 25%, note that) that suit have, which also provides good explosion resistance. You also nuked pierce resistance to the default armor vest values. More than half of fauna deals pierce\slash or just brute damage. And here we have tiny buff to slash resistance to offset all those nerfs. No fun allowed.

Y'all salty you can't instacrit salvage tech guy with china because he managed to find one hardsuit off a wreck or what? More and more cargo PRs feels like "imded" PRs...

Reply:
_Protection from explosions is absolutely not a must..._

You are not listening. Miners work with explosives - thus explosive resistance. Actual explosive resistance. It's in their line of work. You are contradicting it by saying that our salvage techs don't work with dangerous explosives, and because of that suit doesn't need it. It's a miner hardsuit. Not a spationaut one. And there is dangerous explosives that salvage techs meet (see mines). Mines is a lethal hazard when you are not equipped for it. Miners are equippred for blasts. Thus, their gear supposed to have proper damage resistance.

_Salvagers are generally fighting AI enemies..._

Movespeed is everything. Especially for those, who fight against AI hordes, that straight up rush you. You either tank it with big armor numbers (not in this case), or kite it with movespeed (not in this case either). So in the end we have shit armour and shit movespeed. Cool.

_I actually tested this ingame believe it or not..._

Honestly - ain't believing that. Also to mention new mob, that shitspawn lesser copies of it - good luck with that in miner hardsuit (Emo almost died in a cap vest trying to kill it, without 25% movespeed penalty [add magboots penalty on top] and with better defense stats).

_My face when the 25 tc weapon needs to use up 75% of its intentionally very very powerful super death ammo just to crit a single guy..._

Then dont? Why would you use your precious very very powerful super death ammo on target that resistant to it? Pull out a python and kill slow-ass salv with AP, no? _Oh, but hey, we want to have universal solution in a form of click-dead weapon._ Too bad when you can't remove people just with one click, too bad. Time for a new PR.


Yet again - it's a not roundstart item. It's a salv loot and it's supposed to be a good loot. We don't have MAXIM hardsuit yet, so it's a peak salvage protection for now.

Reply:
Talking to a walls never been so fun before.

Salvagers =\= miners. Their hardsuit - their hardsuit. Miner hardsuit - different brach of work, with different specifications. Salvager will die from touching mine, miner - won't.

Yet again i mention HORDE of mobs. You can't kite em all while moving like turtle. That's when defense compensation kicks in.

Nukies are more than equipped for any station combat scenario, especially in war ops. China lake shouldn't be universal answer for everything. And don't even compare their defense values with mining hardsuit lmao. 10% MS debuff for minimum 50% resist to everything - yeah nah. Syndiecomm suit comes with no movespeed penalty and shitton of resistances. Aside from being more vulnerable to explosives than elite or jugg - it's the best suit out there.

And yeah, this random rockbreaker wears 100 kilo hardsuit on his back and that gives him protection. Goofy-ahh-tech armor suit doesn't come with broken back penalties.

And yeah, as i mentioned above - AP rounds. If nukies don't use em - their fault. For some reason crew must pull a rocket launcher out of their ass to win, but nukies can just shrug off all bullshit by their sheer amount of TC. Cool.

I won't continue on this one, since you are hellbent on this imded pr and we are just straight up wasting our time here - maintainers will decide.